apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts

```diff
@@ -1090,3 +1091,12 @@ export const useNativeAssetInvalidation = ({networkId, policy, name}: NativeAsse
     invalidate: mutation.mutate,
   }
 }
+
+export const useCreateGovernanceTx = (wallet: YoroiWallet) => {
+  const mutationFn = (certificates: Certificate[]) => {
+    return wallet.createUnsignedGovernanceTx(certificates)
+  }
+
+  const mutation = useMutation({mutationFn, useErrorBoundary: true, retry: false})
+  return {...mutation, createUnsignedGovernanceTx: mutation.mutateAsync}
```

Comment:
the aliases should be on mutationFn, the mutateAsync is wrapped by query, it should consume from mutation.mutateAsync on the client only, yet, be aware that none of the status flag will reflect -> isLoading, isError, while using it.